vite.config.js

```diff
@@ -44,12 +45,18 @@ export default defineConfig(({ mode }) => ({
 			output: {
 				manualChunks: (id) => {
 					if (id.includes("node_modules")) {
+						if (id.includes("react-bootstrap") || id.includes("@restart")) {
+							return "vendor__react-bootstrap";
+						}
+
```

Comment:
WOOOO!! Great job continuing from where we left off Brianna!! I'm amazed by your research skills  - as always - and am so happy that you found this fix after reading the Vite and rollup documentation.

Reply:
You really sprung it into action with finding the manual chucking commented out worked again!